util/topgen.py

```diff
@@ -730,7 +730,7 @@ def main():
 
         # Create filtered list
         filter_list = [
-            module['name'] for module in topcfg['module']
+            module['type'] for module in topcfg['module']
```

Comment:
oops! sorry, i'm pretty sure I created this bug. 
How did this manifest btw?

Reply:
I found this as I put the `_aon` suffix to those modules in AON domain :)

Reply:
I think I made this bug. though...